### PR TITLE
Adds linting rules to all files including tests and test utils

### PR DESCRIPTION
**Pull request by Moejoe90**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -12,7 +12,9 @@
     "jsx": "react",
     "sourceMap": true,
     "typeRoots": ["./node_modules/@types", "./@types"],
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "noUnusedLocals": true,
```
Comment (desfero) [inline tsconfig.json]: 🎖

File: app/test-e2e/wallets/WalletMigrate.spec.e2e.ts [hunk 2]
```diff
@@ -1,17 +1,22 @@
-import { Q18 } from "../../config/constants";
-import { getNonceRpc, ETransactionStatus } from "../utils/ethRpcUtils";
-import { createAndLoginNewUser } from "../utils/userHelpers";
-import { tid } from "../utils/selectors";
+import BigNumber from "bignumber.js";
 import web3Accounts from "web3-eth-accounts";
-import { getChainIdRpc, sendRawTransactionRpc, getTransactionReceipt } from "../utils/ethRpcUtils";
+import { appRoutes } from "../../components/appRoutes";
```
Comment (desfero) [inline app/test-e2e/wallets/WalletMigrate.spec.e2e.ts]: should be with our own imports, I'm wondering whether it's worth to enforce this by some linting rule?

File: app/test-e2e/eto/InvestWithUpgradedWalletEurT.spec.e2e.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
-import { createAndLoginNewUser } from "../utils/userHelpers";
-import { tid } from "../utils/selectors";
 import { INV_ETH_EUR_ICBM_HAS_KYC } from "../constants";
-import { confirmAccessModal, LONG_WAIT_TIME } from "../utils/index";
 import { etoFixtureAddressByName } from "../utils";
+import { confirmAccessModal, LONG_WAIT_TIME } from "../utils/index";
```
Comment (desfero) [inline app/test-e2e/eto/InvestWithUpgradedWalletEurT.spec.e2e.ts]: can be combine with the above one

File: app/test-e2e/eto/InvestWithUpgradedWalletEther.spec.e2e.ts [hunk 4]
```diff
@@ -1,8 +1,8 @@
-import { createAndLoginNewUser } from "../utils/userHelpers";
-import { tid } from "../utils/selectors";
 import { INV_ETH_EUR_ICBM_HAS_KYC } from "../constants";
-import { confirmAccessModal } from "../utils/index";
 import { etoFixtureAddressByName } from "../utils";
+import { confirmAccessModal } from "../utils/index";
```
Comment (desfero) [inline app/test-e2e/eto/InvestWithUpgradedWalletEther.spec.e2e.ts]: can be combined with the above one

File: app/test-e2e/eto/InvestWithRegularEther.spec.e2e.ts [hunk 5]
```diff
@@ -1,8 +1,8 @@
-import { confirmAccessModal } from "./../utils/index";
-import { createAndLoginNewUser } from "../utils/userHelpers";
-import { tid } from "../utils/selectors";
 import { INV_EUR_ICBM_HAS_KYC_SEED } from "../constants";
 import { etoFixtureAddressByName } from "../utils";
+import { tid } from "../utils/selectors";
+import { createAndLoginNewUser } from "../utils/userHelpers";
+import { confirmAccessModal } from "./../utils/index";
```
Comment (desfero) [inline app/test-e2e/eto/InvestWithRegularEther.spec.e2e.ts]: can be combined with the one on line `2`

File: app/test-e2e/wallets/WalletMigrate.spec.e2e.ts (hunk 2 above)
Comment (Moejoe90) [reply app/test-e2e/wallets/WalletMigrate.spec.e2e.ts]: If there is a linter rule

**Pull request merged**